flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/handlers/TaskManagersHandler.java

```diff
@@ -39,6 +42,7 @@
 
 	private final FiniteDuration timeout;
 
+	private static final String taskManagerIDKey = "taskmanagerid";
```

Comment:
This is referencing your addition in WebRuntimeMonitor. You can use this constant there as well. I would furthermore use the constant field style a la "TASKMANAGER_ID_KEY".